website/content/tools/autoscaling/agent.mdx

```diff
@@ -13,13 +13,12 @@ multiple configuration files or directories to configure the agent.
 
 ## Nomad Namespaces
 
-The Nomad Autoscaler currently has limited support for
-[Nomad Namespaces][nomad_namespaces]. The `nomad` configuration below supports
-specifying a namespace; if configured with a namespace, the Autoscaler will
-retrieve scaling policies and perform autoscaling only for jobs in that
-namespace. If the special wildcard namespace value `*` is used, the Autoscaler
-agent will retrieve scaling policies from all namespaces. A future version will
-include support for multiple namespaces.
+The Nomad Autoscaler agent's [`nomad`][autoscaler_agent_nomad] configuration
+block supports specifying a [Nomad namespace][nomad_namespaces]. If configured
+with a namespace, the Autoscaler will retrieve scaling policies and perform
+autoscaling only for jobs in that namespace. If the special wildcard namespace
+value `*` is used, the Autoscaler agent will retrieve scaling policies from all
+namespaces. A future version will include support for multiple namespaces.
```

Comment:
active voice option, if you'd like:

```suggestion
You can configure Nomad Autoscaler to use a [Nomad namespace][nomad_namespaces]
using the agent's [`nomad`][autoscaler_agent_nomad] configuration block,
so the Autoscaler retrieves scaling policies and perform autoscaling only for
jobs in that namespace. If you use the special wildcard namespace value `*`,
the Autoscaler agent will retrieve scaling policies from all namespaces.
A future version will include support for multiple namespaces.
```

Reply:
I'd probably break that first sentence in half? Keep "...agent's configuration supports..." (which is fine) and change the second half to "If you configure..." What do you think of that?

Reply:
Thanks for the suggestion Daniel!
Active voice and should use present tense... but I didn't bother for this one (bad me). At some point, I'll update the content to match our style guide.